test/libsolidity/lsp/didOpen_with_import.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
```

Comment:
Actually, the LSP does generate a warning for not having this pragma in your source file, which IMHO makes sense to keep. It would be just inconsistent otherwise (IMHO). So removing this line does actually affect diagnostics for any events below. I soft-vote for keeping it. @cameel :)

Reply:
Oh, right these are not executed by isoltest.

But I really think we should not have hard-coded concrete versions in tests. How about using `pragma solidity *;`?